bin/zopen-build

```diff
@@ -383,6 +383,7 @@ Option:
   --oci                 build and publish an OCI image to \$ZOPEN_IMAGE_REGISTRY.
   -s                    exec a shell before running configure. Useful when
                         manually building ports.
+  --signPax, -sp        This option signs the pax file.
```

Comment:
I noticed we use dashes in most cases to seperate words in our options. We should probably be consistent.
```
--sign-pax
```
@DevonianTeuchter , not sure if you have a convention already that we should follow?